lib/printer.ts

```diff
@@ -844,6 +871,9 @@ function genericPrintNoParens(path, options, print) {
     case "DirectiveLiteral": // Babel 6
         return fromString(nodeStr(n.value, options));
 
+    case "InterpreterDirective":
+        return fromString(`#!${n.value}\n`);
```

Comment:
```suggestion
        return fromString(`#!${n.value}\n`, options);
```
The potential values of `n.value` seem diverse enough to potentially be affected by the options.